definitions/region/connections.yaml

```diff
@@ -0,0 +1,243 @@
+- Connections:
+    - Austria
+    - Belgium
+    - Bosnia and Herzegovina
+    - Bulgaria
+    - Croatia
+    - Czechia
+    - Denmark
+    - EMPIRE V1.0.0/v51|North Sea|Borssele
+    - EMPIRE V1.0.0/v51|North Sea|Dogger Bank
+    - EMPIRE V1.0.0/v51|North Sea|East Anglia
+    - EMPIRE V1.0.0/v51|North Sea|Firth of Forth
+    - EMPIRE V1.0.0/v51|North Sea|Helgolander Bucht
+    - EMPIRE V1.0.0/v51|North Sea|Hollandsee Kust
+    - EMPIRE V1.0.0/v51|North Sea|Hornsea
+    - EMPIRE V1.0.0/v51|North Sea|Moray Firth
+    - EMPIRE V1.0.0/v51|North Sea|Nordsoen
+    - EMPIRE V1.0.0/v51|North Sea|Norfolk
+    - EMPIRE V1.0.0/v51|North Sea|Outer Dowsing
+    - EMPIRE V1.0.0/v51|North Sea|Sorlige Nordsjo I
+    - EMPIRE V1.0.0/v51|North Sea|Sorlige Nordsjo II
+    - EMPIRE V1.0.0/v51|North Sea|Utsira Nord
+    - EMPIRE V1.0.0/v51|Norway|Norgemidt
+    - EMPIRE V1.0.0/v51|Norway|Ostland
+    - EMPIRE V1.0.0/v51|Norway|Sorland
+    - EMPIRE V1.0.0/v51|Norway|Troms
+    - EMPIRE V1.0.0/v51|Norway|Vestmidt
+    - Estonia
+    - Finland
+    - France
+    - Germany
+    - Greece
+    - Hungary
+    - Ireland
+    - Italy
+    - Latvia
+    - Lithuania
+    - Luxembourg
+    - Netherlands
+    - North Macedonia
+    - Poland
+    - Portugal
+    - Romania
+    - Serbia
+    - Slovakia
+    - Slovenia
+    - Spain
+    - Sweden
+    - Switzerland
+    - United Kingdom
+
```

Comment:
The countries and EMPIRE-regions are already defined in other yaml files, only the connections should be included in this file.

```suggestion
```